portfolio/src/main/webapp/script.js

```diff
@@ -30,3 +30,21 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
+/** 
+    Sets tab buttons display content 
```

Comment:
It's great to document functions this way. But this description is hard to understand. Maybe something like "Switches the view to the given tab." WDYT?

Reply:
Rereading that description it seems like it's barely English, thanks! I switched it to "Shows the content of the active tab"